static/css/explore.scss

```diff
@@ -66,26 +66,31 @@
 .search-section {
   align-items: center;
   display: flex;
-  flex-direction: row;
-  margin: 20px 0;
+  flex-direction: column;
+  flex-shrink: 0;
+  height: 79px;
+  margin: 24px 0 60px 0;
+  width: 100%;
 
   .btn {
     height: 2.5rem;
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
   }
 
-  .experiment-tag {
-    border-radius: 10px;
-    border-style: solid;
-    border-width: 3px;
-    color: var(--blue);
-    display: flex;
-    float: left;
-    font: normal 0.85em monospace;
-    padding: 4px 6px;
-    height: 80%;
-    margin-right: 20px;
+  .early-preview-tag {
+    color: var(--gm-3-ref-secondary-secondary-30, #004a77);
+    font-family: Google Sans;
```

Comment:
Reviewers: Using "Google Sans" throughout for now until base.scss can be updated. Let me know if there's a different value to prefer or if I should remove font-family entirely.